src/server/worldserver/worldserver.conf.dist

```diff
@@ -1207,6 +1207,14 @@ Event.Announce = 0
 
 BeepAtStart = 1
 
+#
+#    FlushAtStart
+#        Description: Flush when the world server finished starting.
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+
+FlushAtStart = 1
```

Comment:
It should be "flash" here and wherever "flush" is used

Reply:
What is flush? Guessing for consistency should be used same word in this code.